usdQt/hierarchyModel.py

```diff
@@ -151,7 +151,7 @@ def __OnObjectsChanged(self, notice, sender):
                     indexPath = indexPrim.GetPath()
 
                     for resyncedPath in resyncedPaths:
-                        commonPath = resyncedPath.GetCommonPrefix(path)
+                        commonPath = resyncedPath.GetCommonPrefix(resyncedPath)
```

Comment:
No, in my haste I picked the wrong path.  It should be indexPath.  Let me try and get a better test case that'll catch this...